tests/putil/test_clockobject.py

```diff
@@ -16,13 +29,13 @@ def test_clock_jump_frame_time(clockobj):
 def test_clock_get_real_time(clockobj):
     current_time = clockobj.get_real_time()
     time.sleep(0.4)
-    assert clockobj.get_real_time() - current_time >= 0.39
+    assert math.isclose(clockobj.get_real_time() - current_time, 0.4, abs_tol=CLOCK_INACCURACY)
 
 
 def test_clock_get_long_time(clockobj):
     current_time = clockobj.get_long_time()
     time.sleep(0.4)
-    assert clockobj.get_long_time() - current_time >= 0.39
+    assert math.isclose(clockobj.get_long_time() - current_time, 0.4, abs_tol=CLOCK_INACCURACY)
```

Comment:
The reason this was a `>=` is because on a build server under load it's always possible for the CPU scheduler to switch to another, higher-priority process in the middle of executing this code.  It can therefore take quite a lot longer in reality.

Note that pytest also has an approx function.  I don't know whether one has more advantage than the other.

Reply:
You're right about that. I replaced math.isclose with explicit calculations instead.